paper-input-container.html

```diff
@@ -201,8 +202,8 @@
       .input-content.label-is-floating ::content .paper-input-label {
         -webkit-transform: translateY(-75%) scale(0.75);
         transform: translateY(-75%) scale(0.75);
-        -webkit-transition: -webkit-transform 0.25s;
-        transition: transform 0.25s;
+        -webkit-transition: -webkit-transform 0.25s, width 0.26s;
```

Comment:
The idea was to make sure the scaling was finished before the width was finished.
I don't think it was necessary so I made them the same. In my mind the scaling and width will occur incrementally together and be OK. Didn't notice any issues retesting after this change.